src/styles/_utilities.scss

```diff
@@ -1,3 +1,16 @@
+:root {
+  // toolbar-height is needed in various places in defining layouts.
+  --gi-header-height: #{$headerHeightMobile};
```

Comment:
This css-variable will be useful in solving many layout related issues in the future, so created it while working on this issue.